### DISCUSSION TITLE
Packaging changes for external collectors in our native DEB/RPM packages.

### DISCUSSION THREAD

**Discussion by Ferroin** [netdata/netdata#15116] [Announcements]
As [previously outlined on our blog](https://blog.netdata.cloud/split-plugin-packages/), we are splitting external data collection plugins to their own packages in our native DEB/RPM packaging repositories.

Users are encouraged to read the blog post linked above for details on what’s changing and what they may need to do in response on their systems.

This change will be in our nightly builds starting from 2023-05-31 (versions newer than v1.39.0-73-nightly), and our stable builds starting from v1.40.0. This only affects users who are using our native DEB or RPM packages. If you are instead using our Docker images, static builds, or a local build, you should be completely unaffected.

**Comment by Mworth62** [comment]
I AM PLEASED WITH OUTCOME  LETS HOPE WE ARE ALL OF THE SAME OPPINION,
